### ISSUE TITLE
super projekt

### ISSUE THREAD

**Issue by dziobex** [dziobex/iq-increaser#1]
super

**Comment by dziobex** [comment]
racja, bardzo super

**Comment by dziobex** [comment]
macie bardzo racje, jest super

**Comment by dziobex** [comment]
dzięki

**Comment by dziobex** [comment]
chciało by się powiedzieć: zajebisty!

**Comment by dziobex** [comment]
> chciało by się powiedzieć: zajebisty!

dziena!

**Comment by dziobex** [comment]
> sponka projekt!

no prawda!!! świetnie zrobiony

**Comment by dziobex** [comment]
> > sponka projekt!
> 
> no prawda!!! świetnie zrobiony

dzięki panowie! dzięki!

**Comment by JanJon5000** [comment]
koniec tego narcyzmu!

**Comment by dziobex** [comment]
> koniec tego narcyzmu!

chyba nie dodaliśmy takiego modułu w tym **rewelacyjnym** projekcie! czemu mielibyśmy się wycofać z niego!

**Comment by dziobex** [comment]
> > koniec tego narcyzmu!
> 
> chyba nie dodaliśmy takiego modułu w tym **rewelacyjnym** projekcie! czemu mielibyśmy się wycofać z niego!

nie, chyba takiego nie ma, ale brzmi super czyli idealnie dla tego jakże super projektu!

**Comment by dziobex** [comment]
świetny ten projekt

**Comment by dziobex** [comment]
e p i c k i !

**Comment by dziobex** [comment]
> świetny ten projekt

dzięki bardzo

**Comment by dziobex** [comment]
fajny, muszę przyznać, jest ten projekt

**Comment by dziobex** [comment]
> taki nie za słodki

dzięki ja nie lubię cukrzycy

**Comment by dziobex** [comment]
**_spoko projekt naprawde działa_**

**Comment by JanJon5000** [comment]
kurcze to naprawde funckjonuje!

**Comment by JanJon5000** [comment]
ale niezle!

**Comment by JanJon5000** [comment]
niezly programero!

**Comment by dziobex** [comment]
dzieki

**Comment by JanJon5000** [comment]
super!

**Comment by dziobex** [comment]
fajne, kiedy update?

**Comment by dziobex** [comment]
> fajne, kiedy update?

nie wiem, chyba idzie to do archiwizacji

**Comment by dziobex** [comment]
> > fajne, kiedy update?
> 
> nie wiem, chyba idzie to do archiwizacji

ale jak to! rozwijaj ten super projekt

**Comment by dziobex** [comment]
> > > fajne, kiedy update?
> > 
> > 
> > nie wiem, chyba idzie to do archiwizacji
> 
> ale jak to! rozwijaj ten super projekt

nie chce mi sie

**Comment by dziobex** [comment]
> > > > fajne, kiedy update?
> > > 
> > > 
> > > nie wiem, chyba idzie to do archiwizacji
> > 
> > 
> > ale jak to! rozwijaj ten super projekt
> 
> nie chce mi sie

:(
